charts/sscs-bulk-scan/values.yaml

```diff
@@ -6,8 +6,6 @@ java:
       resourceGroup: sscs-bulk-scan
```

Comment:
Can't comment on line 2 but can you remove the https please, 
it should be:
`hmctspublic.azurecr.io/sscs/bulk-scan:latest`

otherwise it won't work by default